main.js

```diff
@@ -10,32 +10,38 @@ var currentPalette = [];
 var savedPalettes = [];
 
 // Event Listeners
-window.addEventListener('load', function() {
+window.addEventListener('load', function () {
 	generateRandomPalette();
 	updatePalette(currentPalette)
 });
 
-newPaletteBtn.addEventListener('click', function() {
+newPaletteBtn.addEventListener('click', function () {
 	generateRandomPalette();
 	updatePalette(currentPalette)
 });
 
-savePaletteBtn.addEventListener('click', function() {
+savePaletteBtn.addEventListener('click', function () {
 	savePalette();
 	displaySavedPalettes();
 	console.log(savedPalettes);
 });
 
-miniPaletteContainer.addEventListener('click', function(event) {
-	var target = event.target;
-	if (target.parentNode.classList.contains('delete-button')) {
-			var index = target.parentNode.dataset.index;
-			deleteSavedPalette(index);
-			console.log(savedPalettes);
+miniPaletteContainer.addEventListener('click', function (event) {
```

Comment:
Nice work adding the dataset.index functionality in order to be able to pull the index position later on in the code! This made deleting saved palettes much easier.

Reply:
Yes, in the displaySavedPalette function we assign a value to the data-index property and we are able to access that value in the event listener.